Unexpected log field

## Bug Report
```
Error: 4/06 05:50:15.661 +00:00] [ERROR] [pd_service_discovery.go:221] ["[pd] failed to update member"] [error="[PD:client:ErrClientGetMember]get member failed%!(EXTRA []string=[http://127.0.0.1:32879/ http://127.0.0.1:37741/ [http://127.0.0.1:41645])](http://127.0.0.1:41645]%29/)"]
Error: 4/06 05:50:15.661 +00:00] [ERROR] [pd_service_discovery.go:221] ["[pd] failed to update member"] [error="[PD:client:ErrClientGetMember]get member failed%!(EXTRA []string=[http://127.0.0.1:32879/ http://127.0.0.1:37741/ [http://127.0.0.1:41645])](http://127.0.0.1:41645]%29/)"]
```

The log is not well readable by misusing `FastGenByArgs` which is introduced by https://github.com/tikv/pd/pull/6046

--- a/client/pd_service_discovery.go
+++ b/client/pd_service_discovery.go
@@ -218,7 +218,7 @@ func (c *pdServiceDiscovery) updateMemberLoop() {
 			failpoint.Continue()
 		})
 		if err := c.updateMember(); err != nil {
-			log.Error("[pd] failed to update member", errs.ZapError(err))
+			log.Error("[pd] failed to update member", zap.Strings("urls", c.GetURLs()), errs.ZapError(err))
 		}
 	}
 }
@@ -453,7 +453,7 @@ func (c *pdServiceDiscovery) updateMember() error {
 		// the error of `switchTSOAllocatorLeader` will be returned.
 		return errTSO
 	}
-	return errs.ErrClientGetMember.FastGenByArgs(c.GetURLs())
+	return errs.ErrClientGetMember.FastGenByArgs()
 }
 
 func (c *pdServiceDiscovery) getClusterInfo(ctx context.Context, url string, timeout time.Duration) (*pdpb.GetClusterInfoResponse, error) {

--- a/tests/integrations/tso/server_test.go
+++ b/tests/integrations/tso/server_test.go
@@ -153,8 +153,8 @@ func (suite *tsoServerTestSuite) TestConcurrentlyReset() {
 	for i := 0; i < 2; i++ {
 		go func() {
 			defer wg.Done()
-			for i := 0; i <= 100; i++ {
-				physical := now.Add(time.Duration(2*i)*time.Minute).UnixNano() / int64(time.Millisecond)
+			for j := 0; j <= 100; j++ {
+				physical := now.Add(time.Duration(2*j)*time.Minute).UnixNano() / int64(time.Millisecond)
 				ts := uint64(physical << 18)
 				suite.resetTS(ts, false, false)
 			}

